app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,18 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
+      attr_writer :jwt_identity
+      attr_reader :authentication_input, :decoded_token
```

Comment:
I think you should expose here the authentication input fields (service_id, account ...) that we are using so access to them be more straight forward 

For example for whom that will use this class will have `AuthenticationParameters.service_id `instead of `AuthenticationParameters.authentication_input.service_id`

Reply:
Done